src/patterns/ec2-app/base.ts

```diff
@@ -182,6 +194,66 @@ export interface GuEc2AppProps extends AppIdentity {
    * discouraged) to limit to a subset of the available subnets.
    */
   publicSubnets?: ISubnet[];
+
+  /**
+   * Configure Google Auth.
+   */
+  googleAuth?: {
+    /**
+     * Enables Google Auth (via Cognito). **Additional MANUAL steps required -
+     * see below.**
+     *
+     * Limits access to members of the allowed Google groups.
+     *
+     * Note, this does not currently support simultaneous machine access, so
+     * only set to true if you only require staff access to your service, or are
+     * supporting machine access in some other way.
+     *
+     * MANUAL STEPS: to get this to work, we need a Google Project and
+     * associated credentials. Full instructions can be found here:
+     *
+     * https://docs.google.com/document/d/1_k1FSE52AZHXufWLTiKTI3xy5cGpziyHazSHTKrYfco/edit?usp=sharing
+     *
+     * DevX hope to automate this process in the near future.
+     */
+    enabled: true;
+    /**
+     * The domain users will access your service.
+     *
+     * Set this to the same as for certificateProps.
+     */
+    domain: string;
+    /**
+     * Groups used for membership checks.
+     *
+     * If specified, cannot be empty. Users must be a member of at least one
+     * group to gain access.
+     *
+     * @defaultValue [`engineering@theguardian.com`]
```

Comment:
TODO - I think these actually need to use the `guardian.co.uk` variant so let's update the default value and docs here to make this clear.